doc/nrf/ug_nrf9160.rst

```diff
@@ -101,10 +162,59 @@ If the modem is shut down gracefully before the next boot (by using ``AT+CFUN=0`
 
 For more detailed information, see the `system mode section in the AT Commands reference document`_.
 
+.. _nrf9160_ug_fota:
+
+FOTA upgrades
+*************
+
+You can upgrade the firmware of the nRF9160 over the air, thus without a wired connection.
+Such an upgrade is called a FOTA (firmware over-the-air) upgrade.
+FOTA upgrades can be used to apply delta patches to the `LTE modem`_ firmware and to replace the upgradable bootloader or the application.
+
+.. note::
+   Even though the Secure Partition Manager and the application are two individually compiled components, they are treated as a single binary blob in the context of firmware upgrades.
+   When we refer to the application in this section, we therefore mean the application including the Secure Partition Manager.
+
+A FOTA upgrade requires the following steps:
+
+1. Make sure that your application supports FOTA upgrades.
+      To download and apply FOTA upgrades, your application must use the :ref:`lib_fota_download` library.
+      This library deduces the type of upgrade by inspecting the header of the firmware and invokes the :ref:`lib_dfu_target` library to apply the firmware upgrade.
+      By default, the DFU target library supports all kinds of FOTA upgrades, but you can disable support for specific targets.
+
+      In addition, the following requirements apply:
+
+      * If you want to upgrade the application, :doc:`mcuboot:index` must be used as upgradable bootloader (:option:`CONFIG_BOOTLOADER_MCUBOOT`).
+      * If you want to upgrade the upgradable bootloader, the :ref:`bootloader` must be used (:option:`CONFIG_SECURE_BOOT`).
+      * If you want to upgrade the modem firmware, neither MCUboot nor the immutable bootloader are required, because the modem firmware upgrade is handled by the modem itself.
+
+#. Create a binary file that contains the new image.
+      This step does not apply for upgrades of the modem firmware.
+      You can download delta patches for the modem firmware from the `nRF9160 product website (compatible downloads)`_.
+
+      To create a binary file for an application upgrade, make sure that (:option:`CONFIG_BOOTLOADER_MCUBOOT`) is enabled and build the application as usual.
+      The build will create several binary files (see :ref:`mcuboot:mcuboot_ncs`).
+      The :file:`app_update.bin` file is the file that should be uploaded to the server.
+
+      To create a binary file for a bootloader upgrade, make sure that (:option:`CONFIG_SECURE_BOOT`) is enabled and build MCUboot as usual.
+      The build will create a binary file that contains two variants of the upgradable bootloader, one for each bootloader slot.
```

Comment:
a zip file?

Reply:
How does CONFIG_MCUBOOT_BUILD_S1_VARIANT come in here?

Reply:
It will *not* generate a zip file. 

Enabling `CONFIG_MCUOBOOT_BUILD_S1_VARIANT` will build one or more `.bin` files which are upgrade candidates linked against the other mcuboot slot.

Reply:
If `CONFIG_MCUBOOT_BUILD_S1_VARIANT` is enabled. the build will create an upgrade candidate binary file for the bootloader linked against the S1 slot.